internal/config/types.go

```diff
@@ -24,11 +26,22 @@ type ScriptStep struct {
 	ExpectPermissionship string `yaml:"expectPermissionship"`
 	NumExpected          uint   `yaml:"numExpected"`
 	Updates              []Update
+	Checks               []Check
 	Schema               string
 	Consistency          string
 	Context              *ProtoStruct
 }
 
+// Check is one of a set of Checks handed to CheckBulk
+type Check struct {
+	Resource             string
+	Subject              string
+	Permission           string
+	Context              *ProtoStruct
+	ExpectNoPermission   bool   `yaml:"expectNoPermission"`
+	ExpectPermissionship string `yaml:"expectPermissionship"`
+}
```

Comment:
This is a subset of the ScriptStep, but I don't know that there's a good way to reuse them.